docs/developer/v1alpha1-compared-to-v1alpha2.md

```diff
@@ -0,0 +1,66 @@
+# Cluster API v1alpha1 compared to v1alpha2
+
+## Providers
+
+### v1alpha1
+
+Providers in v1alpha1 wrap behavior specific to an environment to create the infrastructure and bootstrap instances into
+Kubernetes nodes. Examples of environments that have integrated with Cluster API v1alpha1 include, aws, gcp, openstack,
+azure, vsphere and others. The provider vendors in Cluster API's controllers, registers their actuators with the
+controllers and runs a single manager (with the Cluster API controllers) to complete a Cluster API management cluster.
```

Comment:
```suggestion
controllers, and runs a single manager (with the Cluster API cluster and machine controllers) to complete a Cluster API management cluster.
```

Reply:
Should we mention that there are other controllers from Cluster API that providers usually don't vendor, and that are run from the cluster-api manager pod? (MachineDeployment, MachineSet, NodeRef)

Reply:
I'm a little torn, I omitted that detail since I don't think it's terribly important to understand the differences since it's not really different now, but definitely interested to hear why you think we should include it

Reply:
If this is about differences, I think it's fine to omit.